Knowledge/QA glossary.md

```diff
@@ -46,12 +51,18 @@ The glossary is meant to help you get familiar with words and phrases commonly u
 - **Cookies** - are files created by websites you visit. They make your online experience easier by saving browsing information. With cookies, sites can keep you signed in, remember your site preferences, and give you locally relevant content.
 - **CopyClip** - simple and efficient clipboard manager for your Mac.
 - **Crashlytics** - a part of Firebase; used for getting crash/error logs from mobile apps.
-- **CTA (Call to Action)** - this usually refers to a button or other UI element which prominently invites the user to take action on a website or in an app.
+- **CTA (Call to Action)** - Usually refers to a button or other UI element which prominently invites the user to take action on a website or in an app
+Examples: “Sign-up for our newsletter”, “Share on social media”, “Sign-up”, “Get started”, “add to cart”, “Buy now” and so on.
```

Comment:
I know it's a small typo, but can you please put a capital letter on “add to cart”?